upgrade/1.2/Stage_3.md

```diff
@@ -1,13 +1,40 @@
 # Stage 3 - CSM Service Upgrades
 
-**IMPORTANT:**
+1. Install latest document RPM package and prepare assets:
```

Comment:
What ticket is this for?  This doesn't seem to be for either CASMINST-4389 or CASMINST-4388.

Reply:
I found the added commit for CASMINST-4391.  Please add that to the PR description to make it clearer.

Reply:
I just noticed that in Stage 2.3, we copy the docs-csm-latest rpm from m001 to m002.  Wouldn't it flow better if we just install the rpm at that point?

Reply:
actually, i need to remove rpm install, cause we had that installed on m002 when we upgraded it. so that is totally unnecessary